update-tests/src/test/java/io/zeebe/test/ContainerState.java

```diff
@@ -18,13 +18,14 @@
 import java.util.regex.Pattern;
 import net.jodah.failsafe.Failsafe;
 import net.jodah.failsafe.RetryPolicy;
+import org.junit.jupiter.api.extension.ExtensionContext.Store.CloseableResource;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.testcontainers.containers.ContainerFetchException;
 import org.testcontainers.containers.ContainerLaunchException;
 import org.testcontainers.containers.Network;
 
-public class ContainerState {
+public class ContainerState implements CloseableResource {
```

Comment:
In junit5, `CloseableResource` objects which are put in a store are automatically closed when the test lifecycle ends (so if it's on a parameter or field, when the test ends; if it's on a static field, when the class/suite ends).